src/main/java/frc/robot/subsystems/shooter/Roller.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.shooter;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Roller extends SubsystemBase {
+  /** Creates a new Roller. */
+  public Roller() {}
```

Comment:
this is not what i meant by separating shooter and rollers. This seems somewhat unnecessary to me rn.